Gulpfile.js

```diff
@@ -4,29 +4,40 @@ const gulp = require('gulp');
 const sass = require('gulp-sass');
 
 const paths = {
-  scss: [
-    './core/**/*.scss',
-    './contrib/*.scss',
-  ]
+  markup: {
+    src: './contrib/**/*.html',
+  },
+  styles: {
+    src: [
+      './contrib/**/*.scss',
+      './core/**/*.scss',
+    ],
+    dest: './contrib/',
+  }
 };
 
-gulp.task('serve', ['sass'], function() {
+function styles() {
+  return gulp.src(paths.styles.src)
+    .pipe(sass({includePaths: bourbon}))
+    .pipe(gulp.dest(paths.styles.dest));
+};
+
+function serve(done) {
   browserSync.init({
     open: false,
-    server: './contrib',
+    server: './contrib/',
   });
+  done();
+}
 
-  gulp.watch(paths.scss, ['sass']);
-  gulp.watch('contrib/*.html').on('change', browserSync.reload);
-});
+function reload(done) {
+  browserSync.reload();
+  done();
+}
 
-gulp.task('sass', function() {
-  return gulp.src(paths.scss)
-    .pipe(sass({
-      includePaths: ['styles'].concat(bourbon),
-    }))
-    .pipe(gulp.dest('./contrib'))
-    .pipe(browserSync.stream());
-});
+function watch() {
+  gulp.watch(paths.markup.src, reload);
+  gulp.watch(paths.styles.src, gulp.series(styles, reload));
```

Comment:
does the default task series need to include `styles` if it's being run as part of `watch` here?
Personally I have `watch` set up to run any build tasks and then `serve`, but that's just me.

Reply:
I realize now that this call to `gulp.series` probably doesn't run when `watch` is called initially, but only if a file changes.